suite-native/module-home/src/components/Assets.tsx

```diff
@@ -28,9 +35,65 @@ type HomeAssetsNavigationProp = TabToStackCompositeNavigationProp<
     RootStackParamList
 >;
 
+interface AssetType {
+    symbol: NetworkSymbol;
+    network: NetworkSymbol;
+    assetBalance: BigNumber;
+    assetFailed: boolean;
+    fiatBalance: string;
+}
+
 export const Assets = () => {
     const navigation = useNavigation<HomeAssetsNavigationProp>();
     const { applyStyle } = useNativeStyles();
+    const accounts = useSelector(selectAccounts);
+    const coins = useSelector(selectCoins);
+
+    const assets: { [key: string]: Account[] } = useMemo(() => ({}), []);
+    accounts.forEach(account => {
+        if (!assets[account.symbol]) {
+            assets[account.symbol] = [];
+        }
+        assets[account.symbol].push(account);
+    });
+
+    const networks = Object.keys(assets);
+
+    const assetsData: AssetType[] = useMemo(
```

Comment:
This whole thing + also previous stuff `assets` etc. could be easily moved into selector.